lib/rexml/parsers/baseparser.rb

```diff
@@ -115,6 +115,7 @@ class BaseParser
       def initialize( source )
         self.stream = source
         @listeners = []
+        @scanner = StringScanner.new('')
```

Comment:
How about `@attributes_scanner`? Because this is used only for parsing attributes.

Reply:
OK, I see.
I will change it.